tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -63,88 +50,86 @@ def test_positive_create_with_name(setup, name):
 
     :parametrized: yes
     """
-    compresource = entities.LibvirtComputeResource(
-        location=[setup.loc],
+    name = gen_string('alphanumeric')
+    description = gen_string('alphanumeric')
+    display_type = 'spice'
+    libvirt_url = 'qemu+tcp://libvirt.example.com:16509/system'
+
+    cr = module_target_sat.api.LibvirtComputeResource(
         name=name,
-        organization=[setup.org],
-        url=setup.current_libvirt_url,
+        description=description,
+        provider=FOREMAN_PROVIDERS['libvirt'],
+        display_type=display_type,
+        organization=[module_org],
+        location=[module_location],
+        url=libvirt_url,
     ).create()
-    assert compresource.name == name
+    assert cr.name == name
+    assert cr.description == description
+    assert cr.provider == 'Libvirt'
```

Comment:
Should this be 
```suggestion
    assert cr.provider == FOREMAN_PROVIDERS['libvirt']
```
I know it's the same thing, but I think it's better than hard coding the check for `'Libvirt'`

Reply:
updated